### ISSUE TITLE
photometric errors overestimated by a factor of ~1.7-1.9

### ISSUE THREAD

**Issue by djones1040** [genghisken/ps1#163]
Hi Ken, I think this might be something Mark has to work on, but recording it as a bug at least.  The chi^2 of pre-explosion epochs for SNe in YSE is approximately 0.3-0.35 (looking at test cases of 2020juq, 2020ioz, 2020ivg, 2020inp).  This translates to overestimated errors by nearly a factor of two, and from looking at photpipe reductions of these SNe this appears to apply to both the regular photometry and the forced photometry.  Here's one example file:
![photcomp](https://user-images.githubusercontent.com/1512537/84206194-80f14080-aa63-11ea-99f8-52c9d486bf8f.png)

This is most likely hurting our discovery rates at high redshift by up to a factor of three, so potentially a pretty critical bug to fix.

**Comment by mhuber0** [comment]
Yeah, this is a problem for Mark and have added an IPP ticket for it. If you send me the photopipe (and might as well IPP) photometry I can do an internal check and some IPP tests.

**Comment by djones1040** [comment]
sent via email!  Thanks for taking a look.
